src/applications/disability-benefits/686c-674/tests/config/veteranInformation.unit.spec.jsx

```diff
@@ -1,71 +1,22 @@
 import React from 'react';
-import sinon from 'sinon';
+import { render } from '@testing-library/react';
 import { expect } from 'chai';
-import { mount } from 'enzyme';
 import { Provider } from 'react-redux';
-import { DefinitionTester } from 'platform/testing/unit/schemaform-utils.jsx';
+import { getFormDOM } from 'platform/testing/unit/schemaform-utils.jsx';
 import createCommonStore from 'platform/startup/store';
 import VeteranInformationComponent from '../../config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js';
-import formConfig from '../../config/form.js';
 
 const defaultStore = createCommonStore();
 
 describe('<VeteranInformationComponent />', () => {
   it('Should Render', () => {
-    const wrapper = mount(
+    const wrapper = render(
       <Provider store={defaultStore}>
         <VeteranInformationComponent />
       </Provider>,
     );
+    const formDOM = getFormDOM(wrapper);
 
-    expect(wrapper.find('div.usa-alert-info')).to.exist;
```

Comment:
Decided to remove a number of these unit tests because they weren't actually testing anything in this component.